omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -26,7 +26,7 @@
 	<!-- Annotation based controllers -->
 	<bean class="org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping"/>
 	
-	<context:component-scan base-package="org.openmrs.module.registrationcore.web.controller" />
```

Comment:
The @Component annotation in the searchHandler wasn't working without that and there is no package org.openmrs.module.registrationcore.web.controller in this project or am I missing something ?